app/views/bartender/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<p id="notice"><%= notice %></p>
+
+<body class="text-center">
+  <h2 class= "text-center"> <%= @bartender.name %> </h2>
+
+  <h4 class= "text-center"> Location <%= @bartender.location %> </h4>
+
+  <h4 class= "text-center">
+    Languages:
+    <%= @bartender.loc %>
+
+
+
+
+  </h4>
+
+  <p class= "text-center"> <%= @bartender.bio %> </p>
+
+  <p class="text-center"> </p>
+
+
+
+
+  <% @cocktail.doses.each do |dose| %>
+  <ul class ="text-center">
```

Comment:
I will then create another branch with the correct views and overwrite this branch.